src/SIM/AdaptiveSIM.C

```diff
@@ -212,6 +212,7 @@ bool AdaptiveSIM::adaptMesh (int iStep, std::streamsize outPrec)
 
 bool AdaptiveSIM::writeGlv (const char* infile, int iStep)
 {
+  this->savePoints(0.0, iStep);
```

Comment:
The writeGlv method should strictly deal with VTF-output only. Is there somewhere else you can put that savePoints call?

Reply:
sure, I put it in the solveStep method instead.